includes/lib/usage-tracking/class-usage-tracking-base.php

```diff
@@ -320,13 +320,14 @@ public function send_usage_data() {
 			return;
 		}
 
-		$usage_data = call_user_func( $this->callback );
+		$this->send_event( 'send_usage_data' );
+		$this->send_event( 'system_log', $this->get_system_data() );
```

Comment:
Are we concerned with `system_log` timing out? If not, I wonder if we should move it before `send_usage_data` so that we can compare the `send_usage_data` timestamp to `sensei_stats_log` to get a better picture of how long it's taking to run.

Reply:
Can we compare the `sensei_system_log` timestamp to the `sensei_stats_log` timestamp if we're curious about that?

Reply:
Yup, that should work.